cmd/gcp-pd-csi-driver-operator/main.go

```
@@ -0,0 +1,51 @@
+func main() {
+	command := NewOperatorCommand()
+	code := cli.Run(command)
+	os.Exit(code)
+}
+
+func NewOperatorCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gcp-pd-csi-driver-operator",
+		Short: "OpenShift GCP PD CSI Driver Operator",
+		Run: func(cmd *cobra.Command, args []string) {
+			cmd.Help()
```

golangci-lint (2.12.2)

[error] Error return value of `cmd.Help` is not checked

(errcheck)